ethereum/core/src/test/java/org/hyperledger/besu/ethereum/privacy/PrivateStateGenesisAllocatorTest.java

```diff
@@ -146,8 +146,8 @@ private void assertGenesisAccountApplied() {
   private void assertManagementContractApplied() {
     Account managementProxy = worldState.get(FLEXIBLE_PRIVACY_PROXY);
     assertThat(managementProxy.getCode()).isEqualTo(PROXY_RUNTIME_BYTECODE);
-    assertThat(managementProxy.getStorageValue(UInt256.ZERO))
-        .isEqualTo(UInt256.fromBytes(DEFAULT_FLEXIBLE_PRIVACY_MANAGEMENT));
+    assertThat(managementProxy.getStorageValue(Bytes32.ZERO))
+        .isEqualTo(Bytes32.leftPad(Bytes.wrap(DEFAULT_FLEXIBLE_PRIVACY_MANAGEMENT)));
```

Comment:
DEFAULT_FLEXIBLE_PRIVACY_MANAGEMENT is of type Address, which implements Bytes. Why do we need the wrap?

Reply:
We don't, removing it.